### DISCUSSION TITLE
《CMU 10-414 deep learning system》学习笔记 | 周鑫的个人博客

### DISCUSSION THREAD

**Discussion by giscus** [LittleHeroZZZX/zhouxin-space#115] [Announcements]
# 《CMU 10-414 deep learning system》学习笔记 | 周鑫的个人博客

Lecture 1: Introduction and Logistics 课程的目标 本课程的目标是学习现代深度学习系统，了解包括自动微分、神经网络架构、优化以及 GPU 上的高效操作在内的技术的底层原理。作为实践

https://www.zhouxin.space/notes/notes-on-cmu-10-414-deep-learning-system/

**Comment by kukutu** [comment]
求更新
